docs/document/docs/guides/tabs-customzation.md

```diff
@@ -0,0 +1,200 @@
+---
+id: tabs-customization
+title: Tabs customization
+sidebar_label: Tabs customization
+---
+
+## Overview
+This article explains how to utilize and customize the Tabs component.<br>
+We assume the following scenario:
+1. Append KUC components to Tabs content.
+2. Append Kintone native field to Tabs content.
+
+### Components to use
+- [Tabs](../components/desktop/tabs.md)
+- [ReadOnlyTable](../components/desktop/readonly-table.md)
+
+## Completed image
+
+The completed image of the customized page is as follows:
+
+![tabs customize](assets/tabs_customize.gif)
+
+## What you will need to have ready
+
+Create an application that includes two blank fields with IDs 'space' and 'spas', and add three table fields below them with IDs 'sales_record_table' and 'task_management_table', 'sales_order_table'.
```

Comment:
I thought you missed some points and I wanna suggest a better format.
The sentence below will be correct, right?

```suggestion
Create an application that includes the fields as follows:
- Two blank fields with id "space_for_native_kintone" and "space_for_custom_components"
- Three table fields with id "sales_record_table", "task_management_table", and "sales_order_table".
```